static/scss/catalog/body.scss

```diff
@@ -35,6 +48,51 @@ h1, h2, h3, h4, h5 {
   }
 }
 
+.topics {
+  padding-inline-start: 0px;
+  margin-top: 15%;
+  margin-right: 18px;
+  width: 231px;
+
+  @include media-breakpoint-down(md) {
+    display: flex;
+    flex-wrap: wrap;
+    margin-left: 20px;
+    margin-top: 35px;
+    width: fit-content;
+  }
+
+  .topic {
+    height: 70px;
+    width: 220px;
+    text-align: left;
+    padding: 22px 15px;
+    border-style: solid;
+    border-radius: 6px;
+    color: white;
+    font-size: 18px;
+    font-weight: 600;
+
+    @include media-breakpoint-down(md) {
+      height: 50px;
+      width: fit-content;
+      padding: 10px;
+    }
+
+    &.selected {
+      background: #f3f3f7;
+
+      a {
+        color: $primary;
```

Comment:
Overall in xPRO I think we are not underlining texts. I think the same will go here as well. Currently hovering over the topics shows an underline both in selected and unselected cases. Also, there is no underline stying in the design in Invision as well.

<img width="264" alt="image" src="https://user-images.githubusercontent.com/34372316/229477844-4e572ca7-5482-4049-aa67-7bc1c0db07f4.png">

Reply:
Fixed in 228539e6b95de04edb05c01d1d2fe85e813c964b

Reply:
@asadali145 I thought I mentioned but I think I forgot, This same style needs to be applied to Home Page as well e.g. no underline.

<img width="489" alt="image" src="https://user-images.githubusercontent.com/34372316/230305373-45658f12-6f4c-4e75-820f-43eca7fc50c0.png">